Makefile

```
@@ -247,3 +247,16 @@ mv $(1) $(1)-$(3) ;\
 } ;\
 ln -sf $(1)-$(3) $(1)
 endef
+
+.PHONY: verify-mod-tidy
+verify-mod-tidy:
```

Could we add documentation?

```suggestion
verify-mod-tidy: # Ensure that running go mod tidy has a clean exit. 
```